adserver/tasks.py

```diff
@@ -731,6 +731,23 @@ def calculate_ad_ctrs(days=7, min_views=1_000):
             ad.save()
 
 
+@app.task()
+def notify_on_ad_image_change(advertisement_id):
+    ad = Advertisement.objects.filter(id=advertisement_id).first()
+    if not ad or not ad.image:
+        log.warning("Invalid ad passed to 'notify_on_ad_image_change'")
+        return
+
+    ad_url = generate_absolute_url(ad.get_absolute_url())
+    message = f"Ad '{ad.name}' ({ad_url}) had its image changed to {ad.image.url}"
```

Comment:
Is it possible to actually make this a link with markdown? 🤔 

https://api.slack.com/reference/surfaces/formatting#linking-urls

```suggestion
    message = f"Ad '<{ad_url}|{ad.name}> had its image changed to {ad.image.url}"
```

Reply:
We can try it!